src/gutter-container.coffee

```diff
@@ -0,0 +1,131 @@
+{Model} = require 'theorist'
+{CompositeDisposable, Emitter} = require 'event-kit'
+{Gutter} = require './gutter'
+
+# This class encapsulates the logic for adding and modifying the gutters on a
+# TextEditor.
+
+module.exports =
+class GutterContainer extends Model
+  # @param textEditor (required) The TextEditor that this GutterContainer is
```

Comment:
Check out some of our other docs... we use a markdown-based format to indicate function parameters.